R/zzz.R

```diff
@@ -22,7 +22,7 @@ maybe_register_S3_methods <- function() {
 
   for (i in seq_along(tunable_names)) {
     class <- tunable_classes[[i]]
-    s3_register("tune::tunable", class)
+    s3_register("generics::tunable", class)
   }
```

Comment:
Now that you directly register the S3 methods with the `@export` tag, I don't think we need to lazy register them now. You probably don't need `tunable_names` from above either